packages/ndla-ui/src/Article/Article.tsx

```diff
@@ -177,7 +178,7 @@ export const Article = ({
 
   return (
     <div ref={wrapperRef}>
-      <ArticleWrapper modifier={modifier} ref={articleRef}>
+      <ArticleWrapper modifier={modifier} ref={articleRef} data-article="">
```

Comment:
Ser ikkje heilt kva data-article er godt for her.

Reply:
Det er bare for å ha en stabil måte å hente ut selve artikkel-elementet på. Vi bruker det for eksempel til å plassere forklaringer.